snake/snake-challenge-1/rozwiazania/snake_5.js

```diff
@@ -0,0 +1,196 @@
+
+window.onload = () => createSnakeEngine(
+    "snakeCanvas", 
+    x,
+    200,
+    getNextFrame);
+
+
+var ball = {
+    x: 0,
+    y: 0,
+    direction: "down"
+}
+//moje zmienne
+var x = 6; //wielkosc planszy
```

Comment:
skoro ta zmienna reprezentuje wielkosc planszy, to zamiast komentarza lepiej by bylo, jakby jej nazwa mowila to wprost